python/tempo/tsdf.py

```diff
@@ -585,6 +586,41 @@ def resample(self, freq, func=None, metricCols = None, prefix=None, fill = None)
     enriched_tsdf = rs.aggregate(self, freq, func, metricCols, prefix, fill)
     return(enriched_tsdf)
 
+  def interpolate(self, freq: str, func: str, method: str, target_cols: List[str] = None,ts_col: str = None, partition_cols: List[str]=None, show_interpolated:bool = False):
```

Comment:
Can we make the freq/func optional in the case that the user supplies a resampled data frame? So, I'd like to be able to do this: 

       actual_df: DataFrame =simple_input_tsdf.resample(freq = '1 minute', func='min').interpolate(
            method="linear"
        ).df

Reply:
I can get rid of `func`, but `freq` is required so that the interpolation function know what interval to fill the gaps with. Alternatively,  we can store that as a private variable in the TSDF class if a resample has been done on the dataset?